cli/command/container/opts_test.go

```diff
@@ -1019,7 +1019,7 @@ func TestParseEntryPoint(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if len(config.Entrypoint) != 1 && config.Entrypoint[0] != "anything" {
+	if len(config.Entrypoint) != 1 || config.Entrypoint[0] != "anything" {
 		t.Fatalf("Expected entrypoint 'anything', got %v", config.Entrypoint)
 	}
```

Comment:
Given that we already have the `gotest.tools` assert imported in this package (and use it for other tests in this file), we could use that for this test;

```go
func TestParseEntryPoint(t *testing.T) {
	config, _, _, err := parseRun([]string{"--entrypoint=anything", "cmd", "img"})
	assert.NilError(t, err)
	assert.Check(t, is.DeepEqual(config.Entrypoint, []string{"anything"}))
}
```